.github/workflows/pr-e2e-testing.yaml

```
@@ -2,6 +2,12 @@
 
 on:
   workflow_dispatch:
+  inputs:  
```

actionlint (1.7.7)

unknown Webhook event "inputs". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)